INSTALL.rst

```diff
@@ -34,7 +34,7 @@ gcc
 - 4.9 - 7 (if you want to build of Nvidia GPUs, supported compilers depend on your current `CUDA version <https://gist.github.com/ax3l/9489132>`_)
 
   - CUDA 8.0: Use gcc 4.9 - 5.3
-  - CUDA 9.0 - 9.1: Use gcc 4.9 - 5
+  - CUDA 9.0 - 9.1: Use gcc 4.9 - 5.3
```

Comment:
@psychocoderHPC looks in https://github.com/ComputationalRadiationPhysics/alpaka/pull/566 that indeed GCC 5.5 works with CUDA 9.0